save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/kafka/KafkaMsg.kt

```diff
@@ -0,0 +1,11 @@
+package com.saveourtool.save.kafka
+
+/**
+ * kafka message
+ */
+interface KafkaMsg {
+    /**
+     * @property messageId
+     */
+    val messageId: String?
```

Comment:
Id string? why not int?

Reply:
currently it is not used, this is just the most general case